lib/ui_kit/drawer_menu.dart

```diff
@@ -22,65 +24,101 @@ class DrawerMenu extends StatefulWidget {
 }
 
 class _DrawerMenuState extends State<DrawerMenu> {
-  StoreInteraction get _preference => GetIt.instance.get<StoreInteraction>();
+  final AuthService authService = GetIt.instance.get<AuthService>();
+
   @override
   Widget build(BuildContext context) {
     return Drawer(
-      child: Column(
+      child: Stack(
         children: [
-          DrawerHeader(
-            decoration: const BoxDecoration(
-              color: AppColors.darkBlue,
-            ),
-            child: Padding(
-              padding: EdgeInsets.zero,
-              child: Image.asset(
-                'assets/images/logo.png',
-                width: double.infinity,
-                scale: _kScale,
+          Column(
+            children: [
+              DrawerHeader(
+                decoration: const BoxDecoration(
+                  color: AppColors.darkBlue,
+                ),
+                child: Image.asset(
+                  'assets/images/logo.png',
```

Comment:
create a different class like `AppImage`, and extract this `'assets/images/logo.png'` and other paths into this class
AppImage class should be inside **infrastructure** folder